{{cookiecutter.project_name}}/tests/test_token.py

```diff
@@ -172,7 +172,11 @@ def test_mint(token, owner, receiver):
     Create an approved amount of tokens.
     """
     totalSupply = token.totalSupply()
-    assert totalSupply == 1000
+{%- if cookiecutter.premint == 'y' %}
+    assert token.totalSupply() == {{cookiecutter.premint_amount}}
+{%- else %}
+    assert token.totalSupply() == 1000
```

Comment:
`premint == 'n'` in the `Token.vy` code should probably not have the `else` case, instead having initial supply set to `0`, making the else case here `assert token.totalSupply() == 0`

Reply:
make sense